ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -8998,15 +8998,15 @@ static int zend_jit_init_method_call(dasm_State          **Dst,
 		|2:
 	}
 
-	if (!func
+	if ((!func || zend_jit_may_be_modified(func, op_array))
```

Comment:
`func` here comes from static analysis / zend optimizer. In my understanding it's an indication that the call is non polymorphic, since the function could be resolved statically. A guard was not added in this case, with the assumption that `func` could not change. This assumption doesn't hold true across multiple requests.